### ISSUE TITLE
Missing v1.5.2 Release Notes

### ISSUE THREAD

**Issue by DamianSuess** [UXDivers/Gorilla-Player-Beta-Bugs-1.0.0#350]
As a user, I would like to know what is in the release before upgrading our NuGet packages. If we encounter breaking changes, it would be easier for us to inform you properly.

i.e.
* Missing release notes from v1.5.0 to v1.5.2

**Comment by LeoHere** [comment]
@DamianSuess there are not breaking changes. The "release notes" (I added quotes because they are so short that it might be too much to call them release notes) are [here](https://github.com/UXDivers/Gorilla-Player-Support/releases).

Essentially we added support for VS 2019 on Windows and Mac. Actually the 1.5.2 nuget is the same as the 1.5.0. We only update the version number to keep a uniform version among all Gorilla Player components.
